### DISCUSSION TITLE
Not all Regex Manager Presets are compatible with custom datasources

### DISCUSSION THREAD

**Discussion by astellingwerf** [renovatebot/renovate#23986] [Request Help] [answered]
### How are you running Renovate?

Self-hosted Renovate

### If you're self-hosting Renovate, tell us what version of Renovate you run.

36.52.2

### If you're self-hosting Renovate, select which platform you are using.

GitLab self-hosted

### Was this something which used to work for you, and then stopped?

I am trying to get this working for the first time

### Describe the problem

The `matchStrings` of the Regex Manager Presets `regexManagers:dockerfileVersions` and `regexManagers:githubActionsVersions` don't account for the dot character in the names of custom datasources.

### Relevant debug logs

<details><summary>Logs</summary>

```
Copy/paste the relevant log(s) here, between the starting and ending backticks
```

</details>

### Have you created a minimal reproduction repository?

Placeholder value, please select the correct response from the dropdown

**Comment by secustor** [comment, marked as answer]
https://github.com/renovatebot/renovate/pull/24001
